tools/rvn/CommandLineApp.cs

```diff
@@ -51,6 +56,114 @@ public static int Run(string[] args)
             }
         }
 
+        private static void ConfigureSetupPackage()
+        {
+            _app.Command("create-setup-package", cmd =>
+            {
+                cmd.ExtendedHelpText = cmd.Description = "Creates RavenDB setup given setup-params.json";
+                cmd.HelpOption(HelpOptionString);
+                var setupParameters = ConfigureSetupParameters(cmd);
+                var packageOutputFileParameter = ConfigurePackageOutputFile(cmd);
+
+                cmd.OnExecuteAsync(async token =>
+                {
+                    var setupParam = setupParameters.Value();
+                    var packageOutParam = packageOutputFileParameter.Value();
+                    
+                    if (File.Exists(setupParam) == false)
+                        return ExitWithError("Path to setup params has not found", cmd);
+                    
+                    using (StreamReader file = File.OpenText(setupParam))
```

Comment:
CommandLineApp class shouldn't contain code for all the task implementation. This class is about what switches, options do we support and what to run then. Please move all the details of how this is done to a separate class e.g. `SetupPackageConfigurator`.